kong/cmd/utils/prefix_handler.lua

```diff
@@ -378,6 +378,54 @@ local function write_process_secrets_file(path, data)
   return true
 end
 
+local function pre_create_lmdb(conf)
+  local prefix = conf.prefix
+  local lmdb_path = conf.lmdb_environment_path or "dbless.lmdb"
+  local user = string.match(conf.nginx_user or "", "%w+") or "$USER"
+
+  local dir_name
+  if lmdb_path:sub(1, 1) == "/" then
+    dir_name = lmdb_path
+
+  else
+    dir_name = prefix .. "/" .. lmdb_path
+  end
+
+  if pl_path.isdir(dir_name) then
+    return true
+  end
+
+  log.info("LMDB directory '%s' does not exist, " ..
+           "pre-creating with the correct permissions",
+           dir_name)
+
+  local ok, err = pl_path.mkdir(dir_name)
+  if not ok then
+    return nil, "Can not create directory for LMDB " .. dir_name ..
```

Comment:
```suggestion
    return nil, "can not create directory for LMDB " .. dir_name ..
```

We use lower case log messages (with the exception of "LMDB" which is a term itself).